sklearn/ensemble/tests/test_forest.py

```diff
@@ -1228,3 +1229,23 @@ def test_min_impurity_decrease():
             # Simply check if the parameter is passed on correctly. Tree tests
             # will suffice for the actual working of this param
             assert_equal(tree.min_impurity_decrease, 0.1)
+
+
+def test_nestimators_future_warning():
```

Comment:
It might be better to you pytest.parametrize as above instead of the loop, which will run each estimator as a separate test.

Reply:
FYI:

```python
@pytest.mark.parametrize('forest', [RandomForestClassifier(), RandomForestRegressor(),
                                    ExtraTreesClassifier(), ExtraTreesRegressor(),
                                    RandomTreesEmbedding()])
def test_n_estimators_future_warning(estimator):
    ....
    estimator.fit(X, y)
    ....
```

Reply:
Might be better to parametrize with classes,
```py
@pytest.mark.parametrize('forest', [RandomForestClassifier, RandomForestRegressor,
                         [...]
```
then create the corresponding instances inside the test -- this works better for getting a human readable test name with pytest..

Reply:
Fair enough